ui/widgets/build.gradle.kts

```diff
@@ -30,4 +30,5 @@ dependencies {
   implementation(libs.viewbinding.delegate)
   implementation(libs.androidx.appcompact)
   implementation(libs.bundles.androidx.compose)
+  implementation("io.coil-kt:coil-compose:2.3.0") //TODO delete when merge is done
```

Comment:
did you forget it ?

Reply:
This implementation is on the "home carrousel promotions" PR, so I didn't repeat it here. When merging this PR, the changes on the "home carrousel promotions" PR should override this line.